test/cql-pytest/suite.yaml

```diff
@@ -2,3 +2,5 @@ type: Python
 pool_size: 4
 dirties_cluster:
   - test_native_transport
+extra_scylla_config_options:
+  service_levels_interval_ms: 500
```

Comment:
Any particular reason why this should be in test/cql-pytest/suite.yaml and not in test/pylib/scylla_cluster.py with all the rest of the options that we use to configure Scylla? As you can see test/cql-pytest/suite.yaml didn't yet list any options, and I don't think we should start -  I think it's confusing that each option appears in a different file and nobody knows where to find the relevant options.

Please also add the same option to test/cql-pytest/run.py which has another selection of options which are used when running test/cql-pytest/run or test/alternator/run. Thanks.